### DISCUSSION TITLE
Possible to place data/utility files above web root?

### DISCUSSION THREAD

**Discussion by RyanRoberts** [PrivateBin/PrivateBin#1367] [Q&A] [answered]
### Describe the problem/question

As the title asks, is this possible without editing core files? Initially I thought the "Changing the Path" part of the docs explained this but that's just for running it in a subdirectory, right?

Thanks

### Did you use the FAQ section?

- [X] Yes, I have read [the FAQ](https://github.com/PrivateBin/PrivateBin/wiki/FAQ) and I found no solution/answer there.

### What you did?

n/a

### Issue reproducibility

No, I cannot reproduce it on https://privatebin.net.

**Comment by elrido** [comment, marked as answer]
It is absolutely possible, it's how we ship PB by default in the containers and is exactly what is described in the [section of the install doc](https://github.com/PrivateBin/PrivateBin/blob/master/doc/Installation.md#changing-the-path) that you found. It can totally end up being (and should be) outside/above the web root. While we do release the software such that it works instantly as is, I would highly recommend setting this up if you can.

**Reply by RyanRoberts** [reply]
Thanks! I thought that was the case and on a fresh look I've got it working.
